torch/csrc/distributed/rpc/rref_context.h

```diff
@@ -172,6 +174,7 @@ class RRefContext {
   //     in this map. A message of type RREF_USER_ACCEPT will remove the
   //     corresponding RRef from this map.
   std::unordered_map<ForkId, std::shared_ptr<RRef>, ForkId::Hash> pendingUsers_;
+  std::unordered_map<ForkId, std::weak_ptr<RRef>, ForkId::Hash> confirmedUsers_;
```

Comment:
Add a comment about `confirmedUsers_`. Also, note that the comment above is now no longer valid as it says "The following two maps", but is actually referring to pendingUsers_ and pendingChildren_.